libtsuba/src/RDG.cpp

```diff
@@ -615,6 +615,35 @@ katana::RDG::Store(
   return DoStore(handle, command_line, versioning_action, std::move(desc));
 }
 
+namespace {
+
+const std::string empty = "";
+const std::string&
+GetLatestFileName(
+    const std::string& name,
+    const std::vector<katana::PropStorageInfo>& prop_info_list) {
+  auto psi_it = std::find_if(
+      prop_info_list.begin(), prop_info_list.end(),
+      [&](const katana::PropStorageInfo& psi) { return psi.name() == name; });
+
+  if (psi_it == prop_info_list.end()) {
+    return empty;
+  }
+
+  return psi_it->path();
+}
+
+}  // namespace
+
+const std::string&
+katana::RDG::NodePropertyFileName(const std::string& name) {
+  return GetLatestFileName(name, core_->part_header().node_prop_info_list());
+}
+const std::string&
+katana::RDG::EdgePropertyFileName(const std::string& name) {
+  return GetLatestFileName(name, core_->part_header().edge_prop_info_list());
+}
```

Comment:
I'm particularly interested in this interface which I need in some form to merge some LSPG work; I have a version that returns a Uri, but fails if the property isn't clean or absent. If that's acceptable, I can merge #899 an you could use it here; If not I'm happy to use this when it merges.

Reply:
I can go either way.  I return an empty string if the property is dirty or absent.  So merge away if you want.  Currently, I can't build on HEAD due to arrow issues like this.
`In file included from /home/witchel/katana/external/katana/libtsuba/src/RDG.cpp:35:                                                                                                                                   /home/witchel/katana/external/katana/libtsuba/include/katana/ParquetWriter.h:23:34: error: no member named 'PARQUET_2_6' in 'parquet::ParquetVersion'; did you mean 'PARQUET_2_0'?                                            parquet::ParquetVersion::PARQUET_2_6};                                                                                                                                                                                ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~           `

`
/home/witchel/katana/external/katana/libsupport/include/katana/ArrowVisitor.h:259:17: error: no member named 'INTERVAL_MONTH_DAY_NANO' in 'arrow::Type'                                                                  TYPE_CASE(INTERVAL_MONTH_DAY_NANO)                                                                                                                                                                                 ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~  
`